testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/AbstractQuarkusDeployableContainer.java

```diff
@@ -164,6 +164,8 @@ protected List<String> getArgs(Map<String, String> env) {
         if (suiteContext.get().isAuthServerMigrationEnabled()) {
             commands.add("--hostname-strict=false");
             commands.add("--hostname-strict-https=false");
+        } else { // Do not set management port for older versions of Keycloak for migration tests - available since Keycloak ~22
```

Comment:
```suggestion
        } else { // Do not set management port for older versions of Keycloak for migration tests - available since Keycloak 25
```

Reply:
I'm just wondering why I put 22 there :grin: You're right.

Reply:
@vmuzikar Are you ok with merging it as is and amending it later? IMHO, it's not worth executing the whole CI again only due to this.

Reply:
Works for me. 👍